chord_metadata_service/mohpackets/apis/ingestion.py

```diff
@@ -49,103 +49,123 @@
 router = Router()
 
 
-def create_instance(payload, model_cls: Type):
+def create_instances(payloads: List, model_cls: Type):
+    instances = []
     try:
-        instance = model_cls.objects.create(**payload.dict())
+        for item in payloads:
+            instance = model_cls.objects.create(**item.dict())
+            instances.append(instance)
     except Exception as e:
         return JsonResponse(
```

Comment:
does this means that if it encounters a failure with one item, it immediately returns a BAD_REQUEST without trying to ingest the remaining ones? What happens to any non-failing items already processed?

Reply:
1. Yes, it will break. I think if there is a failure, we'd better re-ingest anyway
2. The ingester has to clean it up, since it's already in the db

Reply:
How does the ingester know what to clean up? Or what to fix before re-ingesting? Should katsu return a list of what was ingested, failed, and not tried?

Reply:
- The step to cleaning up is to delete the current broken program.
- The ingester should see an error message that will help identify the issue.
- katsu does return the ingested list when succeeded, the current failure, but not the not tried list.